src/components/navbar/navbar.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react"
```

Comment:
Move this into the index.js file you will create at components/navbar and export it